pyntc/devices/iosxewlc_device.py

```diff
@@ -0,0 +1,90 @@
+"""Module for using a Cisco IOSXE WLC device over SSH."""
+import time
+
+from .ios_device import IOSDevice
+from pyntc.errors import RebootTimeoutError, OSInstallError, InstallModeRequired
+
+INSTALL_MODE_FILE_NAME = "packages.conf"
+
+
+class IOSXEWLCDevice(IOSDevice):
+    """Cisco IOSXE WLC Device Implementation."""
+
+    def _wait_for_device_start_reboot(self):
+        while True:
+            try:
+                self.open()
+                self.show("show version")
+                return
+            except:  # noqa E722 # nosec
+                break
+
+    def _wait_for_device_reboot(self, timeout=5400):
+        start = time.time()
+        while time.time() - start < timeout:
+            try:
+                self.open()
+                self.show("show version")
+                return
+            except:  # noqa E722 # nosec
+                pass
+
+        raise RebootTimeoutError(hostname=self.hostname, wait_time=timeout)
+
+    def install_os(self, image_name, install_mode=False, install_mode_delay_factor=20, **vendor_specifics):
```

Comment:
I would remove `install_mode` option since that is required, and document that in the method docstring.

Reply:
Good idea. Updated.